src/models/DefaultData.js

```diff
@@ -290,6 +357,48 @@ class DefaultData {
     }
   }
 
+  /**
+   * Get the ID of a linked resource.
+   * @param {string} value - Value of the link.
+   * @returns {string} ID of the linked resource.
+   */
+  getComponentIdFromValue(value) {
+    return value;
+  }
+
+  /**
+   * Indicate if type can have a link.
+   * @param {string} type - Component type.
+   * @returns {boolean} True if component can have link otherwise false.
+   */
+  canHaveLink(type) {
+    return this.definitions.links.some(({ sourceRef }) => sourceRef === type);
+  }
+
+  /**
+   * Indicate if type can be linked with another.
+   * @param {string} source - Source type.
+   * @param {string} target - Target type.
+   * @returns {boolean} True if type can be linked to another otherwise false.
+   */
+  canBeLink(source, target) {
```

Comment:
Can be link -> I understand : the element can be a link
Can be linked -> the element can be linked to another
So I would have call this method `canBeLinked` OR maybe just `canLink` (since in param, you define the source and target)

Reply:
done